SzczepionkaWEB/src/app/component/appointment-details/appointment-details.component.ts

```diff
@@ -27,12 +28,25 @@ export class AppointmentDetailsComponent implements OnInit {
     this.appointmentService.getAppointment(this.patientUUID).subscribe(
       data => {
         this.appointment = data;
+        this.replaceStatusDescription();
       }, error => {
         console.log(error);
       }, () => {
       });
   }
 
+  private replaceStatusDescription() {
+    if(this.appointment.appointmentStatus == 'PLANNED') {
```

Comment:
I think that here we can use triple ===. The == operator will compare for equality after doing any necessary type conversions. The === operator will not do the conversion, so if two values are not the same type === will simply return false.